### PR TITLE
feat: setting type of node

### PR DESCRIPTION
**Pull request by aWN4Y25pa2EK**
1. Enables `UDP` 
2. Adds `arabica-11` example

### CODE REVIEW

**Review by smuu — DISMISSED** [review]
LGTM

File: charts/celestia-node/values.yaml [hunk 1]
```diff
@@ -74,6 +74,12 @@ diagnosticMode:
 ## %%MAIN_CONTAINER/POD_DESCRIPTION%%
 ##
 node:
+  ## Select from: bridge, full, light
+  ## Valid values:
+  ## - bridge
+  ## - full
+  ## - light:
```
Comment (smuu) [inline charts/celestia-node/values.yaml:81]: We need to add some validation to ensure that the user sets one of these values.
One idea would be to use this: https://github.com/celestiaorg/helm-charts/blob/ac54d08c18c96ad5c87796c34b8d9e008f1f2fa8/charts/celestia-node/templates/_helpers.tpl#L46

File: charts/celestia-node/templates/statefulset.yaml [hunk 2]
```diff
@@ -207,16 +207,22 @@ spec:
...
+          readinessProbe:
             exec:
               command:
                 - sh
                 - '-c'
                 - >
                   AUTH_TOKEN=$(celestia bridge auth read --node.store=/home/celestia 2>/dev/null);
+                  {{- if eq .Values.node.nodeType "bridge" }}
+                  curl --max-time 0.5 -s -X POST -H "Content-type: application/json" -H "Accept: application/json" -H "Authorization: Bearer $AUTH_TOKEN" -d '{"id":1,"jsonrpc":"2.0","method":"header.SyncWait","params":[]}' "http://localhost:26658";
+                  {{- else if eq .Values.node.nodeType "full" }}
```
Comment (smuu) [inline charts/celestia-node/templates/statefulset.yaml:220]: `LN` has the same probe as `FN`, so we could add it here

File: charts/celestia-node/values.yaml [hunk 3]
```diff
@@ -74,6 +74,12 @@ diagnosticMode:
...
+  nodeType: bridge
```
Comment (smuu) [inline charts/celestia-node/values.yaml:82]: Can we move this to `node.settings.nodeType` as we have all the other values to specify what we run?

Comment (smuu) [inline charts/celestia-node/values.yaml:82]: This should also change the args here: https://github.com/celestiaorg/helm-charts/blob/ac54d08c18c96ad5c87796c34b8d9e008f1f2fa8/charts/celestia-node/values.yaml#L511

This way, we can start different node types without touching the `args` in the values file.

**Review by smuu — CHANGES_REQUESTED** [review]

File: charts/celestia-node/values.yaml (hunk 1 above)
Comment (aWN4Y25pa2EK) [reply charts/celestia-node/values.yaml:81]: great idea!

File: charts/celestia-node/templates/statefulset.yaml (hunk 2 above)
Comment (aWN4Y25pa2EK) [reply charts/celestia-node/templates/statefulset.yaml:220]: noted!

**Comment by aWN4Y25pa2EK** [comment]
- moved `nodeType` to `node.settings` 
- added logic validation for `nodeType` deployment
- added `readinessprobe` to be the same as the one used by the `FN` 
- added anchors to simplify the `values.fies`

**Pull request merged**